apps/kaizen/api/v1/serializers.py

```
@@ -1,17 +1,198 @@
+        if attrs.get("completed", False) and not attrs.get("completed_time"):
+            attrs["completed_time"] = timezone.now()
+
+        if "due_date" in attrs and attrs["due_date"]:
+            if not self.instance and attrs["due_date"] < timezone.now():
+                raise serializers.ValidationError(
+                    {"due_date": "Due date cannot be in the past."}
+                )
+
+        return attrs
+
+    def validate_reminders(self, value):
```

Ruff (0.8.2)

Missing return type annotation for public function `validate_reminders`

(ANN201)